cmd/podman/networks/update.go

```diff
@@ -0,0 +1,58 @@
+package network
+
+import (
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/containers/podman/v4/pkg/domain/entities"
+	"github.com/spf13/cobra"
+)
+
+var (
+	networkUpdateDescription = `Update an existing podman network`
+	networkUpdateCommand     = &cobra.Command{
+		Use:               "update [options] [NAME]",
+		Short:             "update an existing podman network",
+		Long:              networkUpdateDescription,
+		RunE:              networkUpdate,
+		Args:              cobra.MaximumNArgs(1),
+		ValidArgsFunction: completion.AutocompleteNone,
+		Example:           `podman network update podman1`,
+	}
+)
+
+var (
+	networkUpdateOptions entities.NetworkUpdateOptions
+)
+
+func networkUpdateFlags(cmd *cobra.Command) {
+	flags := cmd.Flags()
+
+	addDNSServerFlagName := "add-dns-servers"
+	flags.StringArrayVar(&networkUpdateOptions.AddDNSServers, addDNSServerFlagName, nil, "add network level nameservers")
+	removeDNSServerFlagName := "remove-dns-servers"
+	flags.StringArrayVar(&networkUpdateOptions.RemoveDNSServers, removeDNSServerFlagName, nil, "remove network level nameservers")
+	_ = cmd.RegisterFlagCompletionFunc(addDNSServerFlagName, completion.AutocompleteNone)
+	_ = cmd.RegisterFlagCompletionFunc(removeDNSServerFlagName, completion.AutocompleteNone)
+}
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: networkUpdateCommand,
+		Parent:  networkCmd,
+	})
+	networkUpdateFlags(networkUpdateCommand)
+}
+
+func networkUpdate(cmd *cobra.Command, args []string) error {
+	var (
+		name string
+	)
+	if len(args) > 0 {
+		name = args[0]
+	}
+
+	err := registry.ContainerEngine().NetworkUpdate(registry.Context(), name, networkUpdateOptions)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
Shouldn't we print something on success? The name of the network, maybe?

Reply:
yeah lets print the name